nautobot_ssot/integrations/vsphere/urls.py

```diff
@@ -0,0 +1,26 @@
+"""URL patterns for nautobot-ssot-servicenow."""
+
+from django.urls import path
+from nautobot.apps.urls import NautobotUIViewSetRouter
+
+from . import models, views
+
+router = NautobotUIViewSetRouter()
+router.register("config/vsphere", viewset=views.SSOTvSphereConfigUIViewSet)
+
+urlpatterns = [
+    path(
+        "config/vsphere/<uuid:pk>/changelog/",
```

Comment:
Is there a reason you appear to be adding your own changelog view when we have SSoT logs and ObjectChange logs that should basically be covering this?